Makefile

```diff
@@ -263,3 +263,10 @@ check-goreleaser-tool-check:
 # Check that all the tools are installed.
 .PHONY: check-tools
 check-tools: check-docker-tool-check check-docker-buildx-tool-check check-docker-compose-tool-check check-protoc-tool-check check-golangci-lint-tool-check check-mockgen-tool-check check-goreleaser-tool-check
+
+.PHONY: fmt-local
+fmt-local:
```

Comment:
I was just adding this `fmt` so that people could have the option of running a faster version. But the new `fmt` doesn’t doesn’t break anything so it should be fine if I replaced the old `fmt` with the new one.

I have replaced the old `fmt` with the new `fmt`.